packages/react-components/react-shared-contexts/library/src/CustomStyleHooksContext/CustomStyleHooksContext.ts

```diff
@@ -62,6 +62,7 @@ export type CustomStyleHooksContextValue = Partial<{
   useSpinnerStyles_unstable: CustomStyleHook;
   useSwitchStyles_unstable: CustomStyleHook;
   useTabStyles_unstable: CustomStyleHook;
+  useInteractiveTabStyles_unstable: CustomStyleHook;
```

Comment:
It is awesome that you thought to add this!  I would hold off on adding it to the type until the component is out of preview and public.

Reply:
Done, thanks for the feedback!